src/theme/Layout/index.js

```diff
@@ -18,7 +18,6 @@ import styles from "./styles.module.css";
 export default function Layout(props) {
   const {
     children,
-    noFooter,
     wrapperClassName,
```

Comment:
I don't know if it's related to this or not, but it looks like the footer is jumping up and overlapping content. 

<img width="457" alt="image" src="https://github.com/denoland/deno-docs/assets/22334764/7664443a-01ef-4912-8672-13fb487e328d">
<img width="1036" alt="image" src="https://github.com/denoland/deno-docs/assets/22334764/02f49ea5-f467-4663-947d-18551acc5037">

Reply:
I think this was from a lingering h-screen on the content container which I removed after you probably already started reviewing so see if you can still reproduce as it may be fixed.

Reply:
Yeah I think that must've been it. 👍